src/pages/Admin/ActiveAlert/ActiveAlert.jsx

```diff
@@ -74,34 +91,35 @@ const ActiveAlertDetails = (data) => {
                   </div>
                   <Select
                     className={classes.ActiveAlert__ActiveAlertModalFormSelect}
-                    defaultValue={data.data.shs[0].name}
+                    defaultValue={'data?.data?.active_alert'}
```

Comment:
Are you expecting the value from the backend?

Reply:
yeah... once they update me with the extra data i'll take out the string values

Reply:
please remove it or comment it out then add some default text, it didn't look good during the demo